src/stdJson.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.9.0;
+
+import "Vm.sol";
+
+// Helpers for parsing keys into types.
+library JsonParser {
```

Comment:
```suggestion
library stdJson {
```

Reply:
We currently have `StdJson`, but should be `stdJson` to be consistent with e.g. `stdStorage`. @ZeroEkkusu just double check me on that

Reply:
@mds1 Yes, the filename should be `StdJson.sol` and the library name `stdJson`.